conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--runslow", action="store_true", default=False, help="run slow tests"
+    )
+
+
+def pytest_configure(config):
+    config.addinivalue_line("markers", "slow: mark test as slow to run")
+
+
+def pytest_collection_modifyitems(config, items):
+    if config.getoption("--runslow"):
+        # --runslow given in cli: do not skip slow tests
+        return
+    skip_slow = pytest.mark.skip(reason="need --runslow option to run")
+    for item in items:
+        if "slow" in item.keywords:
+            item.add_marker(skip_slow)
```

Comment:
Is there a reason for the `--runslow` argument? I ran the tests on my machine, and it doesn't seem to be taking long (we have other tests in the codebase, such as the ILP tests, that take longer than these).
What is the use case for not running the tests marked with `@pytest.mark.slow`?

Reply:
It takes a couple of seconds, which is too long IMO. If an ILP test takes this long, it is probably too slow and we should mark it as such.

> What is the use case for not running the tests marked with @pytest.mark.slow?

I'd say all tests should be fast. The main reason to keep tests that are slow would be: (1) Tests are (the best?) documentation. (2) Even though slow tests should not be required for merging, it is nice to have them when one is working on that specific part of the project (more tests are always better, granted).

Reply:
I have kept the runslow.